cmd/mkcgo/generate.go

```diff
@@ -616,3 +610,20 @@ func fnNeedErrWrapper(fn *mkcgo.Func) bool {
 func fnCalledFromGo(fn *mkcgo.Func) bool {
 	return !fn.Variadic() // cgo doesn't support variadic functions
 }
+
+// goSymName returns the Go symbol name for a C symbol name.
+func goSymName(name string) string {
+	if name == "" {
+		return ""
+	}
+	if *private {
+		return strings.ToLower(name[0:1]) + name[1:]
```

Comment:
Hmm, is this func used in a place where `EVP_aes_128_gcm` would become `eVP_aes_128_gcm`? A lowercase prefix might be better to be consistent. (Keeping the case the same as what OpenSSL has in docs seems like it helps readability too, to me.)

Reply:
I suppose the same could be said the other way, although there's only `point_conversion_form_t` in that direction. 🤷